hyperspy/_signals/lazy.py

```diff
@@ -60,6 +60,15 @@
     _logger.info("Dask widgets not loaded (dask >=2021.11.1 is required)")
 
 
+def _get():
+    try:
+        get = dask.threaded.get
+    except AttributeError:  # pragma: no cover
+        # For pyodide
```

Comment:
What about adding a logger entry here?

Reply:
What should the logger entry says?

Reply:
"Dask threaded get is not available in this environment. Falling back to sequential get."?